src/main/java/no/sikt/nva/pubchannels/dataporten/DataportenPublicationChannelClient.java

```diff
@@ -58,12 +61,19 @@ public static PublicationChannelClient defaultInstance() {
     }
 
     @Override
-    public ThirdPartyJournal getJournal(String identifier, String year) throws ApiGatewayException {
-        var request = createFetchJournalRequest(identifier, year);
+    public ThirdPartyPublicationChannel getJournal(String identifier, String year) throws ApiGatewayException {
```

Comment:
I guess in reality, the method here is getChannel(ChannelType type, String identifier, int year)

Reply:
Agree, added enum ChannelType in new commit